src/_gettsim/transfers/erwerbsm_rente.py

```diff
@@ -0,0 +1,358 @@
+from _gettsim.shared import dates_active
+
+
+@dates_active(start="2001-01-01")
```

Comment:
I guess all functions in this module should be decorated with `@dates_active` -- possibly with different dates unless none of this has changed since 2001.